### PR TITLE
LoadLibrary for Las functions and conpty support

### PR DESCRIPTION
**Pull request by bingbing8**

### CODE REVIEW

**Review by bagajjal — APPROVED** [review]

File: contrib/win32/win32compat/tnnet.c [hunk 1]
```diff
@@ -82,14 +83,12 @@ processBuffer(HANDLE handle, char *buf, size_t len, unsigned char **respbuf, siz
 			gbVTAppMode = false;
 
 		/* Console has the capability to parse so pass the raw buffer to console directly */
-		ConRestoreViewRect(); /* Restore the visible window, otherwise WriteConsoleW() gets messy */
```
Comment (manojampalam) [inline contrib/win32/win32compat/tnnet.c:85]: Why is this removed?

File: contrib/win32/openssh/sshd.vcxproj [hunk 2]
```diff
@@ -463,6 +463,7 @@
     <ClCompile Include="$(OpenSSH-Src-Path)contrib\win32\win32compat\win32_sshpty.c" />
```
Comment (manojampalam) [inline contrib/win32/openssh/sshd.vcxproj:463]: Change attributes of conhost_conpty.h, its showing as binary...  also any reason to have this and conhost_conpty.c in win32/OpenSSH? All our Windows specific sources are in win32/win32compat

File: contrib/win32/openssh/conhost_conpty.c [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+		
+		return -1;
+	}
+
+	return 0;
+}
+
+// Return 1 if the resize succeeded, else 0.
+int
+SignalResizeWindow(const HANDLE conhost_pty_sighandle, const unsigned short width, const unsigned short height)
```
Comment (manojampalam) [inline contrib/win32/openssh/conhost_conpty.c]: This is not hooked up to change_window_size request from client. Intentional?

File: contrib/win32/openssh/build.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
       "DockerFile": ".\\contrib\\win32\\openssh\\DockerFile",
       "AdditionalContextFiles": [ ".\\contrib\\win32\\openssh\\OpenSSH-build.ps1" ],
       "DockerImageName": "di-openssh-x64",
-      "BinaryBucket": "OpenSSH-Win64",
+      "BinaryBucket": "OpenSSH-Original",
       "ArtifactsExpected": 1,
```
Comment (manojampalam) [inline contrib/win32/openssh/build.json:15]: Set version to 7.7.1.1.. where are we hosting these changes?

**Review by manojampalam — CHANGES_REQUESTED** [review]
comments inline...

File: contrib/win32/openssh/conhost_conpty.c (hunk 3 above)
Comment (bagajjal) [reply contrib/win32/openssh/conhost_conpty.c]: It's hooked up with window change req from ssh client. This method is called in session_window_change_req().

File: contrib/win32/win32compat/tnnet.c (hunk 1 above)
Comment (bagajjal) [reply contrib/win32/win32compat/tnnet.c:85]: It's not required now.
I added this earlier to mitigate the conpty bug where if we scroll up and start typing then it will print the characters in the scroll up window instead of the last line..  Now conpty has fixed it so this code is not required any more for RS5.

Also this code (if exists) will block any one from maximizing their window as it will restore the window back. Again this is due to the conpty bug which is now fixed.

File: contrib/win32/openssh/build.json (hunk 4 above)
Comment (bagajjal) [reply contrib/win32/openssh/build.json:15]: This is the VSTS artifacts folder name.. If we have the version number then we have to change it everytime when we release a new binary,

Comment (manojampalam) [reply contrib/win32/openssh/build.json:15]: I understand :). I just chose it as a placeholder for version change comment as that file was not included in prior PR.
